docs_sphinx/user/input.rst

```diff
@@ -155,10 +155,21 @@ Poisson inputs
 If the given value for ``rates`` is a constant, then using
 ``PoissonGroup(N, rates)`` is equivalent to
 ``NeuronGroup(N, 'rates : Hz', threshold='rand()<rates*dt')`` and setting the
-group's ``rates`` attribute. If ``rates`` is a string, then it is equivalent
+group's ``rates`` attribute.
+
+If ``rates`` is a string, then it is equivalent
 to ``NeuronGroup(N, 'rates = ... Hz', threshold='rand()<rates*dt')`` with the
 respective expression for the rates (which will be evaluated at every time step
-and therefore allows time-dependent rates). Note that, as can be seen in its
+and therefore allows time-dependent rates, i.e. inhomogeneous Poisson 
+processes). For example, the following code (see also `Timed arrays`_) uses a 
+`TimedArray` to define the rates of `PoissonGroup` as a function of time, 
+resulting in five 100ms blocks of 100 Hz stimulation, followed by 100ms
+silence::
+
+    stimulus = TimedArray([100., 0.] * 5, dt=100.*ms)
```

Comment:
I'd avoid using the "list multiplication" (it certainly has a better name?): in Brian 2 you can write things like `[0, 1]*mV` so using multiplication in a way that does not work for numpy arrays might be confusing. I'd either write it out completely or use `np.tile([100., 0.], 5)`.